sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_client.py

```diff
@@ -0,0 +1,480 @@
+        user_id,  # type: str
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> None
+        """Deletes a user from the Confidential Ledger.
+
+        :param user_id: Identifies the user to delete. This should be an AAD object id or
+            certificate fingerprint.
+        :type user_id: str
+        :return: None
+        :rtype: None
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        if user_id is None:
+            raise ValueError("user_id cannot be None")
+
+        self._client.delete_user(
+            ledger_base_url=self._ledger_url, user_id=user_id, **kwargs
+        )
+
+    @distributed_trace
+    def get_constitution(
+        self, **kwargs  # type: Any
+    ):
+        # type: (...) -> Constitution
+        """Gets the constitution used for governance.
+
+        The constitution is a script that assesses and applies proposals from consortium members.
+
+        :return: The contents of the constitution and its digest.
+        :rtype: ~azure.confidentialledger.Constitution
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_constitution(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return Constitution(script=result.script, digest=result.digest)
+
+    @distributed_trace
+    def get_consortium(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> Consortium
+        """Gets the consortium members.
+
+        Consortium members can manage the Confidential Ledger.
+
+        :return: Details about the consortium.
+        :rtype: ~azure.confidentialledger.Consortium
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_consortium_members(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return Consortium(
+            members=[
+                ConsortiumMember(certificate=member.certificate, member_id=member.id)
+                for member in result.members
+            ]
+        )
+
+    @distributed_trace
+    def get_enclave_quotes(
+        self,
+        **kwargs,  # type: Any
+    ):
+        # type: (...) -> LedgerEnclaves
+        """Gets enclave quotes from all nodes in the Confidential Ledger network.
+
+        :return: Enclave quotes for nodes in the Confidential Ledger.
+        :rtype: ~azure.confidentialledger.LedgerEnclaves
+        :raises: ~azure.core.exceptions.HttpResponseError
+        """
+
+        result = self._client.get_enclave_quotes(
+            ledger_base_url=self._ledger_url, **kwargs
+        )
+        return LedgerEnclaves(
+            {
+                quote.node_id: EnclaveQuote(
+                    node_id=quote.node_id,
+                    mrenclave=quote.mrenclave,
+                    raw_quote=quote.raw,
+                    version=quote.quote_version,
+                )
+                for _, quote in result.enclave_quotes.items()
```

Comment:
nit: 
```python
for quote in result.enclave_quotes.values()
```

`.values()` just gives you the values in a dict

Reply:
ah, brain fart :smile: